src/map.js

```diff
@@ -10,6 +10,8 @@ var Map = React.createClass({
   render() {
     var {number, prevLink, nextLink, ...extraProps} = this.props
     var thumbnail = <img
+        // TODO: Update to pass in an appropriate description of the image instead of just the number
```

Comment:
Unfortunately there didn't seem to be a quick way to get the alt text here. Generally I feel that some alt text is better than nothing.